addons/stock/i18n/stock.pot

```diff
@@ -2926,6 +2926,12 @@ msgstr ""
 msgid "Invalid domain operator %s"
 msgstr ""
 
+#. module: stock
+#: code:addons/stock/models/product.py:0
+#, python-format
+msgid "Invalid domain right operand '%s'. It must be of type Integer/Float"
+msgstr ""
+
 #. module: stock
 #: code:addons/stock/models/product.py:0
 #, python-format
```

Comment:
For future reference, you need to update the existing .pot string that you are changing. But since your fix is wrong you do not need to change it in this case.